deeplearning1/nbs/utils.py

```diff
@@ -54,10 +54,19 @@
 
 
 to_bw = np.array([0.299, 0.587, 0.114])
+
 def gray(img):
-    return np.rollaxis(img, 0, 1).dot(to_bw)
+    if K.image_data_format() == 'channels_last':
```

Comment:
Can you change this to instead check if K.image_dim_ordering() == 'tf'
I belive K.image_data_format() is new with Keras 2 and we are pinned to Keras 1, so this isn't working.